### ISSUE TITLE
How do I interpret the output of license_finder?

### ISSUE THREAD

**Issue by alex4200** [pivotal/LicenseFinder#942]
I have been running `licence_finder` in a virtualenv after I installed a tool called `TMD` with a bunch of dependencies (installed libraries). 

When I run the `license_finder` I get the output:

```
Dependencies that need approval:
TMD, "", unknown
```

I assume the first entry `TMD` means a package (in this case the tool itself), the next entry is the version (none here), and the third entry `unknown` means what?

I thought that tool would find me a license I can use for the tool given the licenses of the dependencies? Maybe this is not what the tool does?

**Comment by exilvm** [comment]
> and the third entry `unknown` means what?

License.

A license of value `unknown` indicates that LicenseFinder was not able to detect it automatically, refer to [this](https://github.com/pivotal/LicenseFinder#setting-licenses) for more information.

**Comment by alex4200** [comment]
@exilvm Yes there is no license yet for "TMD". I want to know what licenses are possible given the dependencies "TMD" relies on. 
Is that possible to do with that tool? Maybe I have the completely wrong tool? 

Does `license_finder` just *list* all licenses?

**Comment by xtreme-shane-lattanzio** [comment]
Hi @alex4200 Im not sure I fully understand your use case but what License Finder does is look through your code for package manager files (go.mod, package.json, build.gradle, etc) If we support you package manager, it will execute logic to find the dependencies installed via one of those files and then output the license for each if it finds one.

In your case, it clearly detected a package manager and inside of the relevant file, TMD is listed. I am not sure what manager you are using so I'm not sure what you are actually doing but the key thing here is that license finder does not assign licences. It will simply report if a given dependency has a license specified in some way.

Hope this helps!
